### PR TITLE
fix(field): Right position different with & whithout fullwidth

### PR DESCRIPTION
**Pull request by sizvix**
The position of the side element should be align on the right of the input.
With or without fullwidth option.

### CODE REVIEW

File: react/Field/Readme.md [hunk 1]
```diff
@@ -84,3 +84,16 @@ const options = [
   />
 </form>
 ```
+
+#### Side element with fullwidth elemeent
```
Comment (gregorylegarec) [inline react/Field/Readme.md]:
```suggestion
#### Side element with fullwidth element
```

File: react/Field/styles.styl [hunk 2]
```diff
@@ -10,10 +10,16 @@
 .o-side
     position absolute
     top 0
```
Comment (gregorylegarec) [inline react/Field/styles.styl]: `top 0` and `left 0` are default values, so I don't think we need to precise them.

**Comment by gregorylegarec** [comment]
@GoOz This solution implies to introduce a new wrapper and introduce potential breaking changes, does it ?

**Comment by GoOz** [comment]
My solution implies no change at all, just using what's already at our disposal. There's no need for another wrapper either.

File: react/Field/styles.styl (hunk 2 above)
Comment (GoOz) [reply react/Field/styles.styl]: That's actually not true. :)
Default `top` position value is the one the element has in the flow, while the `left` one depends on the direction set in the page (sure we use ltr, but still that's not always true)
https://codepen.io/GoOz/pen/Jxogax

my 2 cents.

**Comment by GoOz** [comment]
Ok my mistake, I understood the problem the wrong way. I don't see a better solution so far.

**Comment by ptbrowne** [comment]
Can you upload the styleguide ? It is requested in the PR template for a reason :/

**Comment by gregorylegarec** [comment]
@sizvix Any news ? I am a little bit lost 😅

**Comment by sizvix** [comment]
@ptbrowne @CPatchane , the styleguide is updated   ^^U
Is it OK for you ?

**Comment by ptbrowne** [comment]
Next time, if you can link to the concerned component, it would be nice :) Less work for the reviewer, more chance for the PR to be accepted.

LGTM 👍 

It works for me now, but I am not really happy with the chosen design, if the side element is long or the container is small, the side element can go over the label :/

image

**Comment by nono** [comment]
@gregorylegarec ping can we merge this PR?

**Comment by nono** [comment]
Ping @gregorylegarec

**Pull request merged**